UnityMcpBridge/UnityMcpServer~/src/telemetry.py

```
@@ -0,0 +1,318 @@
+        try:
+            # Load customer UUID
+            if self.config.uuid_file.exists():
+                self._customer_uuid = self.config.uuid_file.read_text().strip()
+            else:
+                self._customer_uuid = str(uuid.uuid4())
+                self.config.uuid_file.write_text(self._customer_uuid)
+                
+            # Load milestones
+            if self.config.milestones_file.exists():
+                self._milestones = json.loads(self.config.milestones_file.read_text())
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)